lib/mongoose_push/service/fcm.ex

```diff
@@ -37,7 +38,9 @@ defmodule MongoosePush.Service.FCM do
 
   @spec push(Service.notification(), String.t(), atom(), Service.options()) ::
           :ok | {:error, term}
-  def push(notification, device_id, worker, opts \\ []) do
+  def push(notification, device_id, _pool, opts \\ []) do
+    worker = Pools.select_worker(:fcm, :prod)
```

Comment:
The second argument here (`mode`) is redundant, as FCM does not support multiple modes.